tests/rt.conf

```diff
@@ -298,113 +302,114 @@ RUN | atmaero_control_p8_rad_micro                      | - noaacloud
+RUN | regional_atmaq_debug                              | - jet gaea derecho s4  noaacloud    | baseline |
 
 COMPILE | atmaq_faster | intel | -DAPP=ATMAQ -DCCPP_SUITES=FV3_GFS_v15p2 -DFASTER=ON -D32BIT=ON | - jet s4 | fv3 |
 RUN | regional_atmaq_faster                             | - jet s4                   | baseline |
 
 ### GNU TESTS ###
 ### CCPP PROD tests ###
-COMPILE | atm | gnu | -DAPP=ATM -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v16_ras,FV3_GFS_v17_p8,FV3_GFS_v17_p8_ugwpv1 | + hera  hercules | fv3 |
-RUN | control_c48                                       | + hera  hercules            | baseline |
-RUN | control_stochy                                    | + hera  hercules            | baseline |
-RUN | control_ras                                       | + hera  hercules            | baseline |
-RUN | control_p8                                        | + hera  hercules            | baseline |
-RUN | control_p8_ugwpv1                                 | + hera  hercules            | baseline |
-RUN | control_flake                                     | + hera  hercules            | baseline |
-
-COMPILE | rrfs | gnu | -DAPP=ATM -DCCPP_SUITES=FV3_RAP,FV3_RAP_sfcdiff,FV3_HRRR,FV3_RRFS_v1beta -D32BIT=ON | + hera  hercules | fv3 |
-RUN | rap_control                                       | + hera  hercules            | baseline |
-RUN | rap_decomp                                        | + hera  hercules            |          |
-RUN | rap_2threads                                      | + hera  hercules            |          |
-RUN | rap_restart                                       | + hera  hercules            |          | rap_control
-RUN | rap_sfcdiff                                       | + hera  hercules            | baseline |
-RUN | rap_sfcdiff_decomp                                | + hera  hercules            |          |
-RUN | rap_sfcdiff_restart                               | + hera  hercules            |          | rap_sfcdiff
-RUN | hrrr_control                                      | + hera  hercules            | baseline |
-RUN | hrrr_control_noqr                                 | + hera  hercules            |          |
-RUN | hrrr_control_2threads                             | + hera  hercules            |          |
-RUN | hrrr_control_decomp                               | + hera  hercules            |          |
-RUN | hrrr_control_restart                              | + hera  hercules            |          | hrrr_control
-RUN | hrrr_control_restart_noqr                         | + hera  hercules            |          | hrrr_control_noqr
-RUN | rrfs_v1beta                                       | + hera  hercules            | baseline |
+COMPILE | atm | gnu | -DAPP=ATM -DCCPP_SUITES=FV3_GFS_v16,FV3_GFS_v16_flake,FV3_GFS_v16_ras,FV3_GFS_v17_p8,FV3_GFS_v17_p8_ugwpv1 | + hera derecho hercules | fv3 |
```

Comment:
So we are running GNU on derecho ? I thought Derecho would not support GNU?

Reply:
We now support both

Reply:
I do not see `ufs_derecho.gnu.lua`. How gnu tests are running on derecho without this file?

Reply:
Yes, @climbfuji @RatkoVasic-NOAA can you please add gnu modulefile for Derecho, if that compiler is now supported?

Reply:
Sure.

Reply:
@climbfuji @zach1221 I added modulefile for Derecho for GNU compiler. Since we don't have GNU compiler installed on Derecho in spack-stack version 1.5.0, I added  it for version 1.5.1.